src/program/graph.rs

```diff
@@ -234,19 +234,30 @@ impl InstructionBlock {
                     Ok(())
                 }
                 InstructionRole::RFControl => {
-                    let frames = match program.get_frames_for_instruction(instruction, true) {
+                    let used_frames = match program.get_frames_for_instruction(instruction, false) {
```

Comment:
[`Option::unwrap_or_default`](https://doc.rust-lang.org/std/option/enum.Option.html#method.unwrap_or_default)

Reply:
Same again for `blocked_frames`